### PR TITLE
add promptpay tel field

### PR DESCRIPTION
**Pull request by jeeholy**
Are name and comment is appropriate?

### CODE REVIEW

File: models/Campground.js [hunk 1]
```diff
@@ -43,6 +43,11 @@ const CampgroundSchema = new mongoose.Schema(
       type: mongoose.Schema.Types.Decimal128,
       required: [true, "Please add a price"],
     },
+
+    promptpayTel: {
+      type: string,
```
Comment (Yiwahpsp) [inline models/Campground.js:48]: Please make it to be "String"

**Review by Yiwahpsp — CHANGES_REQUESTED** [review]

File: models/Campground.js (hunk 1 above)
Comment (FaiiFaiipuipui) [reply models/Campground.js:48]: @jeeholy have you tested anything before requesting a pull request?

I would like to make a clarification here if it's not yet clear for anyone. Only make a pull request when you think the branch is ready to merge. If changes are requested or some minor mistake are overlooked then it's okay. What pull request is **not**__ for is getting other to do the work for you while you haven't done any testing or checking at all.

While it's not explicitly stated, the code on PR are supposed to the very least "seem to work" and if anyone (including me) try to open a PR while the code is obviously not ready, doesn't work, or seem to be carelessly written, then other team member might felt that the person making the PR does not care about contributing to the group work. in the absolute worst case that might result in you getting a 10 points reduction from a score that everyone in the group get (meaning that person will get 10 less point than other people in the group). So let's check our work at least a little bit before opening a pull request.

With respect.

**Review by FaiiFaiipuipui — CHANGES_REQUESTED** [review]
- As yiwah said.
- comment a format in the file and give an example and include a constraint on that format. For example: 0819998888 constraint: regex of number only, length must be 10.

Also,

@jeeholy have you tested anything before requesting a pull request?

I would like to make a clarification here if it's not yet clear for anyone. Please make a pull request only when you think the branch is ready to merge. If changes are requested or some minor mistake are overlooked then it's okay. Pull request is _**not**_ for getting others to do the work for you while you haven't done any testing or checking at all.

While it's not explicitly stated, the code on PR are supposed to the very least "seem to work" and if anyone (including me) try to open a PR while the code is obviously not ready, doesn't work, or seem to be carelessly written, then other team member might felt that the person making the PR does not care about contributing to the group work. in the absolute worst case that might result in that person getting a 10 points reduction from a score that everyone in the group get (meaning that person will get 10 less point than other people in the group). So let's check our work at least a little bit before opening a pull request.

With respect.

**Review by FaiiFaiipuipui — APPROVED** [review]
Very good👍

**Review by tippaphanun — APPROVED** [review]

**Review by Yiwahpsp — APPROVED** [review]
Gooddd

**Pull request merged**